src/styles/Blocks/calendar.scss

```diff
@@ -0,0 +1,27 @@
+.calendar {
```

Comment:
add some indents to avoid this behavior 
<img width="485" alt="Screenshot 2023-03-13 at 18 38 04" src="https://user-images.githubusercontent.com/117587463/224767676-da5e4c38-05db-4f39-9f5e-14065c123781.png">